yt_dlp/extractor/tv5mondeplus.py

```diff
@@ -14,73 +15,69 @@
 
 class TV5MondePlusIE(InfoExtractor):
     IE_DESC = 'TV5MONDE+'
```

Comment:
As the extractor no longer works with TV5MONDE+ this should be renamed.
```suggestion
    IE_DESC = 'TV5MONDE'
```
It is a bit odd to have the extractor still named `TV5MondePlusIE`, but due to the ID space seemingly being the same, I would not rename it.